app/extend/helper.js

```diff
@@ -0,0 +1,7 @@
+const CryptoJS = require('crypto-js');
+
+module.exports = {
+  md5Encrypto(str) {
+    return CryptoJS.MD5(str).toString().toUpperCase();
```

Comment:
The "crypto" package of nodejs already has md5 hash, what's the benefit of "crypto-js"?